frontend/packages/e2e/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I want you to use base config as you would any other package.

https://github.com/liam-hq/liam/blob/350fc44d06a264a5359c842fecc40ab3b73b8806/frontend/packages/erd-core/tsconfig.json#L2

This makes the type test more rigorous and convenient.

Reply:
Sorry for the omission! Corrected.
https://github.com/liam-hq/liam/pull/674/commits/00a1b0a8f5348c4c4b66f9c0c5161bd86653f443